RNA-seq/01-qc_trim_quant.md

```diff
@@ -0,0 +1,244 @@
+	--qualified_quality_phred 15 \
+	--length_required 20 \
+	--report_title "SRR585570" \
+	--json QC/gastric_cancer/fastp_output/SRR585570_fastp.json \
+	--html QC/gastric_cancer/fastp_output/SRR585570_fastp.html
+```
+
+Below, we'll walk through the arguments/options we used to run `fastp`.
+By default, fastp performs adapter trimming, which you can read more about [here](https://github.com/OpenGene/fastp#adapters). 
+For paired-end data like SRR585570, adapters can be detected automatically without specifying an adapter sequence.
+
+#### Input: `-i` and `-I`
+
+These arguments specify the read1 input and read2 input, respectively.
+
+#### fastq output: `-o` and `-O`
+
+These arguments specify the read1 output and read2 output, respectively.
+Note that the output is being placed in `data/fastq/gastric_cancer/SRR585570/`, so the preprocessed FASTQ files will be in the same place as the FASTQ files that have not been filtered.
+There are other ways to organize this and 
+
+#### `--qualified_quality_phred`
+
+[Phred scores](https://en.wikipedia.org/wiki/Phred_quality_score) are the quality information included in a FASTQ file and the values indicate the chances that a base is called incorrectly. Let's look at a screenshot of the Per Base Sequence Quality module from [FastQC bad illumina example](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/bad_sequence_fastqc.html) we linked to above.
+
+![per-base-quality-screenshot](https://user-images.githubusercontent.com/19534205/59292316-d6292500-8c4a-11e9-967b-132bd8c54577.png)
+
+Anything below 20, where a phred score of 20 represents a 1 in 100 chance that the call is incorrect, is considered poor quality by FastQC.
+Using `--qualified_quality_phred 15` (which is the default), means scores >= 15 are considered "qualified." 
+Using the default parameters as we do here, reads will be filtered out if >40% of the bases are unqualified.
+You can read more about the quality filter functionality of fastp [here](https://github.com/OpenGene/fastp/blob/master/README.md#quality-filter).
+
+The Salmon documentation notes that, given the way we run `salmon quant`, quantification may be more sensitive to calls that are likely to be erroneous (of low quality) and, therefore, quality trimming may be important.
+Trimming, in contrast to filtering, refers to removing low quality base calls from the (typically 3') end of reads.
+A recent preprint from the Salmon authors ([Srivastava et al. _bioRxiv._ 2019.](https://doi.org/10.1101/657874)) notes that trimming did not effect mapping rates from random publicly available human bulk (paired-end) RNA-seq samples (they used [TrimGalore](https://github.com/FelixKrueger/TrimGalore)). 
+fastp does have [the functionality](https://github.com/OpenGene/fastp#per-read-cutting-by-quality-score) to perform trimming using a sliding window.
+
+_Note that there are two kinds of encoding for phred scores, phred33 and phred64. 
```

Comment:
If you are going to make this note, can you briefly say what the difference between 33 and 64 is?

Reply:
Or just put a link to something that does explain it.

Reply:
I can put a link.